pkgs/applications/graphics/rendersvg/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv,
+  fetchFromGitHub,
+  rustPlatform,
+  pango,
+  gdk_pixbuf,
+  pkgconfig,
+  qtbase
+}:
+
+rustPlatform.buildRustPackage rec {
+  name = "rendersvg-${version}";
```

Comment:
Please use `pname = "rendersvg";` as `name = "${pname}-${version}";` is already implied.